meta-mender-core/recipes-mender/mender-configure/mender-configure_1.0.0.bb

```diff
@@ -0,0 +1,27 @@
+require mender-configure.inc
+
+################################################################################
+#-------------------------------------------------------------------------------
+# THINGS TO CONSIDER FOR EACH RELEASE:
+# - SRC_URI (particularly "branch")
+# - SRCREV
+# - DEFAULT_PREFERENCE
+#-------------------------------------------------------------------------------
+
+SRC_URI = "git://github.com/mendersoftware/mender-configure-module;protocol=https;branch=1.0.x"
+
+# Tag: TODO
```

Comment:
Still `TODO` ?

Reply:
This is explained in the commit message. Will be fixed before the merge.